src/sentry/static/sentry/app/components/events/interfaces/threads/filter-thread-info.ts

```diff
@@ -0,0 +1,50 @@
+import {Thread, Frame} from 'app/types/events';
```

Comment:
Can we rename these files to be `camelCase` and `tsx` extension to be consistent with the codebase?

Reply:
yes sure 😉 working on it